pandas/tests/tseries/holiday/test_holiday.py

```diff
@@ -193,6 +193,13 @@ def test_holidays_within_dates(holiday, start, expected):
         holiday.dates(utc.localize(Timestamp(start)), utc.localize(Timestamp(start)))
     ) == [utc.localize(dt) for dt in expected]
 
+    
+def test_holidays_outside_dates():
+    hol = Holiday("Dec 25", month=12, day=25, observance=next_monday, days_of_week=(5, 6),
+                  end_date=Timestamp("2013-01-01"))
```

Comment:
The formatting is off, pre-commit should fix this

Reply:
Ran pre-commit, formatting fixed now.